src/components/ClusterNodes/ClusterList.tsx

```diff
@@ -195,28 +241,54 @@ export default function ClusterList({ imageList, isSuperAdmin, namespaceList }:
                         </div>
                     )}
                 </div>
-                <div>{clusterData.nodeCount}</div>
-                <div>
+                <div className="child-shimmer-loading">{clusterData.nodeCount}</div>
+                <div  className="child-shimmer-loading">
                     {errorCount > 0 && (
                         <>
                             <Error className="mr-3 icon-dim-16 dc__position-rel top-3" />
                             <span className="cr-5">{errorCount}</span>
                         </>
                     )}
                 </div>
-                <div className="dc__ellipsis-right">
+                <div className="dc__ellipsis-right child-shimmer-loading">
                     <Tippy className="default-tt" arrow={false} content={clusterData.serverVersion}>
                         <span>{clusterData.serverVersion}</span>
                     </Tippy>
                 </div>
-                <div>{clusterData.cpu?.capacity}</div>
-                <div>{clusterData.memory?.capacity}</div>
+                <div  className="child-shimmer-loading">{clusterData.cpu?.capacity}</div>
+                <div  className="child-shimmer-loading">{clusterData.memory?.capacity}</div>
             </div>
         )
     }
 
-    if (loader) {
-        return <Progressing pageLoader />
+    const renderClusterList = (): JSX.Element => {
+        if (minLoader) {
+            return (
+                <div className="dc__overflow-scroll" style={{ height: 'calc(100vh - 116px)' }}>
+                    <Progressing pageLoader />
+                </div>
+            )
+        } else if (noResults) {
+            return <ClusterNodeEmptyState actionHandler={clearSearch} />
+        } else {
+            return (
+                <div
+                    className="dc__overflow-scroll"
+                    style={{ height: `calc(${showTerminalModal ? '50vh - 125px)' : '100vh - 116px)'}` }}
```

Comment:
Remove Inline css

Reply:
As discussed, we can keep this inline style.